packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -40,7 +40,7 @@ export interface TimelineProps extends ComponentWithClass {
   /**
    * A month will display either side of this start date.
    */
-  startDate?: Date
+  startDate: Date
```

Comment:
This was made required as the component breaks without it.

Reply:
I think the microsite docs will probably also need updating:

https://design-system.digital.mod.uk/timeline

Reply:
Oh yes! This one was already correct on there but some of the other stuff will need updating. Also think the render prop stuff will need updating on there. Will create a new issue...